cypress/integration/marc-authority/marc-authority-view.spec.js

```diff
@@ -0,0 +1,46 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import Features from '../../support/dictionary/features';
+import DevTeams from '../../support/dictionary/devTeams';
+import Permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import DataImport from '../../support/fragments/data_import/dataImport';
+
+describe('MARC Authority management', () => {
+  const userData = {};
+
+  before('Creating user', () => {
+    cy.createTempUser([
+      Permissions.uiQuickMarcQuickMarcBibliographicEditorView.gui,
+      Permissions.inventoryAll.gui,
+    ]).then(createdUserProperties => {
+      userData.id = createdUserProperties.userId;
+      userData.firstName = createdUserProperties.firstName;
+      userData.name = createdUserProperties.username;
+      userData.password = createdUserProperties.password;
+
+      cy.loginAsAdmin();
+      DataImport.uploadMarcBib();
+    });
+  });
+
+  after('Deleting created user', () => {
+    Users.deleteViaApi(userData.id);
```

Comment:
I have found that the instance created by the test is deleted after the test execution. For example the instance with HRID with value in00000000005 is not deleted

Reply:
instance cannot be deleted via ui as far as I know

Reply:
yeah, you can delete it via API

Reply:
InventoryInstance.deleteInstanceViaApi(instanceId);